_maps/DiscStation.dm

```diff
@@ -14,8 +14,10 @@ Names of z-level do not matter, but order does greatly, for instances such as ch
         #include "map_files\generic\z6.dmm"
         #include "map_files\generic\z7.dmm"
 
+
         #define MAP_FILE "DiscStation.dmm"
         #define MAP_NAME "DiscStation"
+        #define MAP_TRANSITION_CONFIG	list("Main Station" = 2,"CentComm" = 1,"Abandoned_Satellite" = 2,"Derelicted Station" = 2,"Mining Asteroid" = 2,"Empty Area 1" = 2,"Empty Area 2" = 2)
```

Comment:
Do you want me to put defines inside defines?
![image](http://i.imgur.com/kykoWDO.jpg)